src/System.Linq/samples/project.json

```diff
@@ -0,0 +1,13 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.10",
```

Comment:
Yeah, I definitely wouldn't want to impede developer workflow.  That is an interesting point you bring up, though.  I wouldn't *think* it would be too much of an issue so long as the base version is the same, mainly because the usage in the samples would be so simple ... but you never know what could happen.  I think it would be a good idea to not use `*` when listing any dependencies.